chapters/data-analysis.tex

```diff
@@ -1,292 +1,487 @@
 %------------------------------------------------
 
 \begin{fullwidth}
-Data analysis is hard. Making sense of a dataset in such a way
-that makes a substantial contribution to scientific knowledge
-requires a mix of subject expertise, programming skills,
-and statistical and econometric knowledge.
-The process of data analysis is therefore typically
-a back-and-forth discussion between the various people
-who have differing experiences, perspectives, and research interests.
-The research assistant usually ends up being the fulcrum
-for this discussion, and has to transfer and translate
-results among people with a wide range of technical capabilities
-while making sure that code and outputs do not become
-tangled and lost over time (typically months or years).
-
-Organization is the key to this task.
-The structure of files needs to be well-organized,
-so that any material can be found when it is needed.
-Data structures need to be organized,
-so that the various steps in creating datasets
-can always be traced and revised without massive effort.
-The structure of version histories and backups need to be organized,
-so that different workstreams can exist simultaneously
-and experimental analyses can be tried without a complex workflow.
-Finally, the outputs need to be organized,
-so that it is clear what results go with what analyses,
-and that each individual output is a readable element in its own right.
-This chapter outlines how to stay organized
-so that you and the team can focus on getting the work right
-rather than trying to understand what you did in the past.
+Transforming raw data into a substantial contribution to scientific knowledge 
+requires a mix of subject expertise, programming skills, 
+and statistical and econometric knowledge. 
+The process of data analysis is, therefore, 
+a back-and-forth discussion between people 
+with differing experiences, perspectives, and research interests. 
```

Comment:
_a back-and-forth discussion between people with differing experiences, perspectives, and research interests._ I think _skill sets_ fits better in this list of characteristics than either _experiences_ or _perspectives_.

Reply:
Changed to skill sets in 97b2ccd.